src/is/L42/flyweight/P.java

```diff
@@ -1,13 +1,25 @@
 // Generated by delombok at Sun Jun 13 15:12:41 NZST 2021
-package is.L42.generated;
+package is.L42.flyweight;
 
+import static is.L42.tools.General.L;
+import static is.L42.tools.General.bug;

```

Comment:
Please, applies the similar comment as expressed for C also in P, I will just discuss new issues here